src/main/java/Orca.java

```diff
@@ -0,0 +1,166 @@
+import java.util.Scanner;
+
+public class Orca {
+    static final String LOGO = "     _______  ______    _______  _______ \n"
+            + "    |       ||    _ |  |       ||   _   | \n"
+            + "    |   _   ||   | ||  |       ||  |_|  | \n"
+            + "    |  | |  ||   |_||_ |       ||       | \n"
+            + "    |  |_|  ||    __  ||      _||       | \n"
+            + "    |       ||   |  | ||     |_ |   _   | \n"
+            + "    |_______||___|  |_||_______||__| |__| \n";
+    static final int MAX_TASKS = 100;
```

Comment:
I like how you named your constants in all capital letters with underscores as spaces